clusterloader2/pkg/util/util.go

```diff
@@ -66,6 +66,11 @@ func GetBool(dict map[string]interface{}, key string) (bool, error) {
 	return getBool(dict, key)
 }
 
+// GetMap tries to return value from map of type map. If value doesn't exist, error is returned.
+func GetMap(dict map[string]interface{}, key string) (map[string]interface{}, error) {
```

Comment:
These changes can be extracted to a separate PR

Reply:
Sure. Would do.

Reply:
PR for the comment above:
https://github.com/kubernetes/perf-tests/pull/1115